src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/HeadersProcessor.scala

```diff
@@ -34,9 +34,6 @@ trait HeadersProcessor extends ToDownloadProcessor with ScorexLogging with Score
 
   val powScheme: PowScheme
 
-  //TODO alternative DDoS protection
```

Comment:
The removed todo should be moved to a place around "keepVersions" field I guess.